src/Styleguide/Elements/RadioGroup.tsx

```diff
@@ -1,25 +1,15 @@
 import React from "react"
-import { BorderProps, SizeProps, SpaceProps } from "styled-system"
 
-import { BorderBox } from "Styleguide/Elements/Box"
-import { Flex } from "Styleguide/Elements/Flex"
-import { Join } from "Styleguide/Elements/Join"
+import { Flex, FlexProps } from "Styleguide/Elements/Flex"
 import { RadioProps } from "Styleguide/Elements/Radio"
-import { Separator } from "Styleguide/Elements/Separator"
 
-export interface RadioGroupProps {
+export interface RadioGroupProps extends FlexProps {
   disabled?: boolean
   onSelect?: (selectedOption: string) => void
   defaultValue?: string
   children: Array<React.ReactElement<RadioProps>>
 }
 
-export interface RadioGroupToggleProps
```

Comment:
Deleted this because it's not used anywhere and doesn't seem to be useful for anything.